src/include/sof/ipc.h

```diff
@@ -204,14 +205,24 @@ int ipc_dma_trace_send_position(void);
 /* get posn offset by pipeline. */
 int ipc_get_posn_offset(struct ipc *ipc, struct pipeline *pipe);
 
+enum ipc_payload_location {
```

Comment:
We add something globally that may not be widely supported. Shouldn't we just guard it with config for `low power ipc` and use it by default when available and fallback to mailbox?